meta-mender-toradex-nxp/recipes-bsp/u-boot/patches/0001-Integration-of-Mender-boot-code-into-U-Boot-v2016.11.patch

```diff
@@ -0,0 +1,49 @@
+From 2013899468926d9b7128314b6d2052105e6858ed Mon Sep 17 00:00:00 2001
```

Comment:
I can't see that you are referring to this patch file (and the next one) anywhere. Are they shadowing the patch files found in meta-mender-core, or did you forget to remove them?

Reply:
I can't figure out where these patches came from, so I removed them, rebased and repushed.